src/lesson5/tv/TvController.java

```diff
@@ -0,0 +1,110 @@
+package lesson5.tv;
+
+import java.util.Scanner;
+
+public class TvController {
```

Comment:
I think TvController can be used with only one TV so makes sense to store Tv as a field and pass it in constructor

Reply:
ok